tests/mitre/test_mitre.sh

```diff
@@ -15,6 +15,8 @@ EXTVARFILE="${MITRE_FILES}/var/linux-external-variables.xml"
 DISTRO="$(cat /etc/*-release | head -n1)"
 DISTRO_NAME="$(cat /etc/*-release | awk '{print $1}' | head -n1)"
 DISTRO_RELEASE="$(cat /etc/*-release | sed -n 's|^[^0-9]*\([0-9]*\).*$|\1|p' | head -n1)"
+SELINUX_ENABLED=$(grep -iqE "(SELINUX=enforcing|SELINUX=permissive)" \
+	/etc/selinux/config 2>/dev/null)
```

Comment:
Fixed in commit 60df8d1. The error was that instead of return value of that grep command I saved its output into SELINUX_ENABLED variable which was always empty because of '-q' cmd option passed to grep.